test/lolcommits/plugin/loltext_test.rb

```diff
@@ -85,6 +85,8 @@ def valid_enabled_config
         # styling overlay
         inputs += %w(true  #2884ae,#7e231f 40)
 
+        inputs += %w(#e96d46 true 23)
```

Comment:
maybe add a 1 line comment above to explain these inputs;

    # border options